xbmc/utils/FileUtils.cpp

```diff
@@ -7,7 +7,6 @@
  */
 
 #include "FileUtils.h"
-
```

Comment:
Revert the removal of this line. We always keep an empty line between header include for the class implemented in the cpp file.